examples/file-sharing/src/main.rs

```diff
@@ -45,7 +45,7 @@
 //!
 //! ## Architectural properties
 //!
-//! - Clean clonable async/await interface ([`Client`]) to interact with the
+//! - Clean clonable async/await interface ([`Client`](network::Client)) to interact with the
```

Comment:
If you wanted, you could split this inner `network` module now into a separate file.